test/spec/ol/format/geojsonformat.test.js

```diff
@@ -524,6 +524,92 @@ describe('ol.format.GeoJSON', function() {
           expect(geojson.features[0].properties.mygeom).to.eql(undefined);
         });
 
+    describe('writing CRS member', function() {
+
+      it('writes null CRS when unknown', function() {
+        var str = JSON.stringify(data),
+            array = format.readFeatures(str);
+        var geojson = format.writeFeatures(array);
+        var result = JSON.parse(geojson);
+        expect(result.crs).to.be(null);
```

Comment:
I don't think this is what we want.  Instead, I think the `crs` member should be absent by default, implying the default CRS for GeoJSON.  The spec does not describe how `crs: null` should be interpreted, and some parsers may only assume the default CRS if the `crs` property is not present.